src/Nethermind/Nethermind.Core/Block.cs

```diff
@@ -122,6 +122,10 @@ public Transaction[] Transactions
 
     [JsonIgnore]
     public ArrayPoolList<AddressAsKey>? AccountChanges { get; set; }
+
+    [JsonIgnore]
+    public long TxByteLength { get; internal set; }
```

Comment:
Can it be used in BlockToProduce instead?

Reply:
Moved